GDJS/Runtime/runtimeobject.js

```diff
@@ -218,6 +231,7 @@ gdjs.RuntimeObject.prototype.onDestroyFromScene = function(runtimeScene) {
  * @return {Object} The internal rendered object (PIXI.DisplayObject...)
  */
 gdjs.RuntimeObject.prototype.getRendererObject = function() {
+    return false;
```

Comment:
`return null;`

Never return a boolean instead of an object. `null` is acceptable for objects.